core/server/api/v2/utils/validators/input/schemas/posts.json

```diff
@@ -0,0 +1,129 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "posts",
+    "title": "posts",
+    "description": "Base posts definitions",
+    "definitions": {
+        "post": {
+            "type": "object",
```

Comment:
@kirrg001, writing some more unit tests for format and other limitations checks, but would be good to have a second pair of :eyes:  if this schema complies with what we expect from data being passed to `/posts` :+1:

Reply:
IMO there is no need to over-optimise the first version. Maybe it should just reflect how it currently works and we will adapt the schema in smaller upcoming PR's? 🤔

Reply:
Makes sense :+1: